src/test/scala/sigmastate/utxo/BasicOpsSpecification.scala

```diff
@@ -272,12 +272,11 @@ class BasicOpsSpecification extends SigmaTestingCommons {
         |}""".stripMargin,
       {
         val data = GetVar(dataVar, dataType).get
-        OR(
-          MapCollection(data,
-            FuncValue(
-              Vector((1, STuple(SByteArray, SLong))),
-              EQ(SelectField(ValUse(1, STuple(SByteArray, SLong)), 2), LongConstant(10)))
-          ).asCollection[SBoolean.type])
+        Exists(data,
```

Comment:
@ergomorphic `Exists` and `ForAll` are back. Made them in http://github.com/ScorexFoundation/sigmastate-interpreter/blob/94fed4472ba0b8ba09f8cea2ad54abefa9816e61/src/main/scala/sigmastate/eval/RuntimeCosting.scala#L955